Squirrely-Ducks/bangazon-sprint-3
# added view for thank you message

## DESCRIPTION
User can now complete an order by adding a payment type to the order

## STEPS TO TEST
1. DROP DATABASE bagazon
2. CREATE DATABASE bangazon
3. node build-db.js
4. npm start
5. search for a product and add to order, then click on shopping cart.
6. click checkout
7. select a payment type, the order should no longer display in shopping cart

## REFERENCE
#16
